streams/src/main/java/org/apache/kafka/streams/Topology.java

```diff
@@ -519,30 +519,32 @@ public synchronized Topology addSink(final String name,
 
     /**
      * Add a new sink that forwards records from upstream parent processor and/or source nodes to Kafka topics dynamically.
+     * The topics that it may ever send to should be pre-created.
      * The sink will use the {@link StreamsConfig#DEFAULT_KEY_SERDE_CLASS_CONFIG default key serializer} and
      * {@link StreamsConfig#DEFAULT_VALUE_SERDE_CLASS_CONFIG default value serializer} specified in the
      * {@link StreamsConfig stream configuration}.
      *
      * @param name              the unique name of the sink
-     * @param topicExtractor    the mapper to dynamically choose the name of the Kafka topic to which this sink should write per reach record
+     * @param topicExtractor    the extractor to determine the name of the Kafka topic to which this sink should write for reach record
```

Comment:
typo `reach` should be `each` (Might be somewhere else, too. Please double check.)